### PR TITLE
Traces 03

### PR DESCRIPTION
**Pull request by marco-tiloca-sics**

### CODE REVIEW

File: draft-ietf-lake-traces.md [hunk 1]
```diff
@@ -1790,35 +1789,35 @@ Then, PRK_2e is calculated using Extract() determined by the EDHOC hash algorith
     PRK_2e = Extract( salt, G_XY ) =
            = HMAC-SHA-256( salt, G_XY )
 
-where salt is the zero-length byte string:
+where salt is TH_2:
 
 ~~~~~~~~
 salt (Raw Value) (0 bytes)
 ~~~~~~~~
```
Comment (malishav) [inline draft-ietf-lake-traces.md:1796]: should this be TH_2 here?

Comment (malishav) [reply draft-ietf-lake-traces.md:1796]: Question: Should TH_2 used as salt be the raw value of the cbor wrapping?

Comment (marco-tiloca-sics) [reply draft-ietf-lake-traces.md:1798]: > should this be TH_2 here?

Indeed. Now fixed, as well as the value of PRK_2e below. All should be ok after that.

Comment (marco-tiloca-sics) [reply draft-ietf-lake-traces.md:1798]: > Question: Should TH_2 used as salt be the raw value or the CBOR wrapping?

My understanding is that it should be the raw value, just like G_XY used as IKM parameter.

Comment (malishav) [reply draft-ietf-lake-traces.md:1798]: Thanks. With the fix above, I confirm the other values and my implementation should be ready for interop :)

**Comment by gselander** [comment]
Merging in preparation for submission.

**Pull request merged**